requirements-common.txt

```diff
@@ -29,3 +29,4 @@ onnxscript<=0.3.1
 openvino
 # for grain_dataset_adapter_test.py
 grain
+datasets
```

Comment:
cannot add this as a dependency for Keras. 
in general if anything is used only for unit tests, you might as well configure in Github actions. I would reccomend using a colab to showcase the test.

Reply:
Agreed as discussed removed the dataset dependency from hugging face

Reply:
removed